partitura/io/importmatch.py

```diff
@@ -1439,55 +265,152 @@ def load_match(
+            line,
+            BaseDeletionLine,
+        ):
             if "leftOutTied" in line.snote.ScoreAttributesList:
                 continue
             else:
-                result.append(dict(label="deletion", score_id=line.snote.Anchor))
-        elif isinstance(line, MatchInsertionNote):
-            result.append(dict(label="insertion", performance_id=line.note.Number))
-        elif isinstance(line, MatchOrnamentNote):
-            if isinstance(line, MatchTrillNote):
+                result.append(dict(label="deletion", score_id=str(line.snote.Anchor)))
+        elif isinstance(
+            line,
+            BaseInsertionLine,
+        ):
+            result.append(dict(label="insertion", performance_id=str(line.note.Id)))
+        elif isinstance(line, BaseOrnamentLine):
+            if isinstance(line, MatchTrillNoteV0):
                 ornament_type = "trill"
+            elif isinstance(line, MatchOrnamentNoteV1):
+                ornament_type = line.OrnamentType
             else:
                 ornament_type = "generic_ornament"
             result.append(
                 dict(
                     label="ornament",
-                    score_id=line.Anchor,
-                    performance_id=line.note.Number,
+                    score_id=str(line.Anchor),
+                    performance_id=str(line.note.Id),
                     type=ornament_type,
                 )
             )
 
     return result
 
 
-# PART FROM MATCHFILE stuff
+# alias
+alignment_from_matchfile = note_alignment_from_matchfile
+
+
+def performed_part_from_match(
+    mf: MatchFile,
+    pedal_threshold: int = 64,
+    first_note_at_zero: bool = False,
+) -> PerformedPart:
+    """
+    Make PerformedPart from performance info in a MatchFile
 
+    Parameters
+    ----------
+    mf : MatchFile
+        A MatchFile instance
+    pedal_threshold : int, optional
+        Threshold for adjusting sound off of the performed notes using
+        pedal information. Defaults to 64.
+    first_note_at_zero : bool, optional
+        When True the note_on and note_off times in the performance
+        are shifted to make the first note_on time equal zero.
+
+    Returns
+    -------
+    ppart : PerformedPart
+        A performed part
 
-def sort_snotes(snotes):
+    """
+    # Get midi time units
+    mpq = mf.info("midiClockRate")  # 500000 -> microseconds per quarter
+    ppq = mf.info("midiClockUnits")  # 500 -> parts per quarter
+
+    # PerformedNote instances for all MatchNotes
+    notes = []
+
+    first_note = next(mf.iter_notes(), None)
+    if first_note and first_note_at_zero:
+        offset = midi_ticks_to_seconds(first_note.Onset, mpq=mpq, ppq=ppq)
+    else:
+        offset = 0
+
+    notes = [
+        dict(
+            id=note.Id,
+            midi_pitch=note.MidiPitch,
+            note_on=midi_ticks_to_seconds(note.Onset, mpq, ppq) - offset,
+            note_off=midi_ticks_to_seconds(note.Offset, mpq, ppq) - offset,
```

Comment:
performedpart notes now contain two extra (optional) fields: note_on_ticks, note_off_ticks. This could be added here as well.